libs/common/types.hpp

```diff
@@ -107,7 +107,10 @@ namespace iroha {
 
     static blob_t<size_> from_string(const std::string &data) {
       if (data.size() != size_) {
-        throw BadFormatException("blob_t: input string has incorrect length " + std::to_string(data.size()));
+        std::string value = "blob_t: input string has incorrect length. Found: "
+            + std::to_string(data.size())
+            + +", required: " + std::to_string(size_);
+        throw BadFormatException(value.c_str());
```

Comment:
It might be better revert that line, i think it was used for debugging (though maybe helpful in future)